utils/api/schemas/collection.ts

```diff
@@ -26,9 +27,9 @@ export const collectionSchema = z.object({
 	viewHash: z.string().nullable(),
 	editHash: z.string().nullable(),
 	metadata: z.record(z.any()).nullable(),
-	kind: z.enum(types.collectionKinds).nullable(),
+	kind: z.enum(collectionKinds).nullable(),
 	doi: z.string().nullable(),
-	readNextPreviewSize: z.enum(types.readNextPreviewSizes).default('choose-best'),
+	readNextPreviewSize: z.enum(readNextPreviewSizes).default('choose-best'),
```

Comment:
Using non-types from `types` really increases the bundlesize, better to manually specify them.